packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -382,8 +384,8 @@ export function TransferPanel() {
       return
     }
 
-    if (!l1Signer || !l2Signer) {
-      throw 'Signer is undefined'
+    if (!isSmartContractWallet && (!l1Signer || !l2Signer)) {
```

Comment:
```suggestion
    if (!isSmartContractWallet && !(l1Signer && l2Signer)) {
```
though they're the same, this seems to make more sense based on the PR description

Reply:
Agree, but would also add this:
```suggestion
    const hasBothSigners = l1Signer && l2Signer
    if (!isSmartContractWallet && !hasBothSigners) {
```